src/mass/adapters/inbound/fastapi_/configure.py

```diff
@@ -18,13 +18,23 @@
 from fastapi import FastAPI
 from ghga_service_commons.api import configure_app
 
+import mass
 from mass.adapters.inbound.fastapi_.routes import router
 from mass.config import Config
 
 
 def get_configured_app(*, config: Config) -> FastAPI:
     """Create and configure a REST API application."""
-    app = FastAPI()
+    doc = mass.__doc__
+    try:
+        title, description = mass.__doc__.split("\n\n", 1)
+    except ValueError:
+        title, description = doc, ""
```

Comment:
Actually, OpenAPI provides three fields: title, summary, description. The title could be the long form of the name, the summary could be what currently is in the docstring. And the description is even more detailed, something that could be taken from a README section. Or it could simply contain a link to the repo.

Will change this.

Reply:
Changed now in ef00a98. Instead of using docstrings, we can use the more public distribution metadata.

Maybe we can put something like this also in the microservice template or service commons.

Reply:
@Cito given the near-identical contents of that function, it would be nice if there was a standardized function for it instead of copy-pasta in every fastapi service.